sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectKey=First-Peoples-Cultural-Council_fv-be
+sonar.organization=first-peoples-cultural-council
```

Comment:
Should these be set in an environment variable?

Reply:
I'm not sure. There is a secret token that I have added to GH secrets which is used for auth I believe. These lines were given by Sonar to be pasted to this config file.
If they pose a security risk, I can add them to GH secrets as well.
@sarahfirstvoices any thoughts on this ?

Reply:
Yes these should be environment vars please. It isn't a security risk, but since this is an open source repo we don't want fpcc-specific config